roles/webview/tasks/webview.yml

```diff
@@ -5,24 +5,81 @@
 # Install
 # +++
 
-- name: copy dist build to source
+- name: create temp unpack location
+  command: "mktemp -d /tmp/ansible.XXXXXXXX"
+  register: temp_webview_location
+
+# FIXME (29-Sept-2016) missing-unarchive-feature
+#       The unarchive `remote_src` option isn't available until Ansible 2.2.
+#       Until then, we will need to make a temp file, use get_url and then use unarchive.
+- name: create temp download location
+  command: "mktemp /tmp/ansible-webview_{{ webview_version }}--XXX.tar.gz"
+  register: temp_webview_tarball_location
+
+- name: "#$%^ing ansible"
+  get_url:
+    url: "https://packages.cnx.org/js-builds/webview_{{ webview_version}}.tar.gz"
+    dest: "{{ temp_webview_tarball_location.stdout }}"
+    force: yes
+
+- name: download and unpack webview
+  become: yes
+  unarchive:
+    # src: "https://packages.cnx.org/js-builds/webview_{{ webview_version}}.tar.gz"
+    # remote_src: yes
+    src: "{{ temp_webview_tarball_location.stdout }}"
+    copy: no
+    # ^^^ changes needed to make this work until the ansible 2.2
+    dest: "{{ temp_webview_location.stdout }}"
+    owner: www-data
+    group: www-data
```

Comment:
Is it not possible to use the url directly?  Like https://github.com/Connexions/cnx-deploy/blob/fabd873dfc07d33ef5be5f79124a08521646865b/tasks/install_python24.yml#L80